services/localfile/localfile.proto

```diff
@@ -275,3 +281,38 @@ message SymlinkRequest {
 message MkdirRequest {
   FileAttributes dir_attrs = 1;
 }
+
+// FileFormat enum what represents file formats
+enum FileFormat {
+  UNKNOWN = 0;
```

Comment:
what's the use case of this? isn't the file format must be known / defined by client when making the request?

Reply:
This is required to backward capabilities. If in new version we will add new type, for sansshell server with old schema version new type would be parsed as UNKNOWN and user get correct error